hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -1234,12 +1240,20 @@ bool MDMParser::getDataUsage(MDM_DataUsage &data)
 
 bool MDMParser::getCellularGlobalIdentity(CellularGlobalIdentity& cgi_) {
     bool result;
```

Comment:
`LOCK()` / `UNLOCK()` needs to be added.

Reply:
fixed